awx/ui_next/src/screens/Host/HostList/HostListItem.jsx

```diff
@@ -13,11 +16,16 @@ import {
 import { Link } from 'react-router-dom';
 import { PencilAltIcon } from '@patternfly/react-icons';
 
-import DataListCell from '@components/DataListCell';
-import DataListCheck from '@components/DataListCheck';
 import { Sparkline } from '@components/Sparkline';
-import VerticalSeparator from '@components/VerticalSeparator';
 import { Host } from '@types';
+import styled from 'styled-components';
+
+const DataListAction = styled(_DataListAction)`
```

Comment:
If we are going to make this custom, I'd rather us be consistent on views and not do this kind of tweak for a specific list item.